csrc/tensor_view.cpp

```diff
@@ -846,13 +846,23 @@ TensorView* TensorView::rFactor(const std::vector<int64_t>& axes) {
         consumer,
         producer);
   } else if (auto grouped_mma = dynamic_cast<GroupedMmaOp*>(definition())) {
+    // I'm not sure how we should handle block scale yet.
+    NVF_CHECK(
+        grouped_mma->outScale() == nullptr &&
+            grouped_mma->outGamma() == nullptr,
+        "not implemented yet");
```

Comment:
@wujingyue BTW, I haven't patched this yet. I think rfactor should propagate through scale as well?!

Reply:
That sounds like a reasonable limitation to keep for now.

> I'm assuming that input matrices are always blocked along the K dimension.

Yeah that sounds right. I think hardware requires input scaling factor to *block* on consecutive memory arrays. At least given the implementation we have for now.

Reply:
At this point we don't know the performance impact of a fused quantization into grouped mm. At least we don't see the sglang kernel outputting lower precisions directly. I kept the option in the API as something to explore when we have implementation plumbed through.